packages/blockchain-extension/cucumber/features/submit.feature

```diff
@@ -34,9 +34,24 @@ Feature: Submit transaction
         And the contract has been instantiated with the transaction '' and args '', not using private data on channel 'mychannel'
         And the gateway 'myGateway' is created
         And I'm connected to the 'myGateway' gateway without association
-        When I submit the transaction 'createConga' with args '["Conga_001", "Big Conga"]'
+        When I submit the transaction 'createConga' on the channel 'mychannel' with args '["Conga_001", "Big Conga"]'
         Then the logger should have been called with 'SUCCESS', 'Successfully submitted transaction' and 'No value returned from createConga'
         Examples:
         | language   | assetType | name               | version |
         | JavaScript | Conga     | JavaScriptContract | 0.0.1   |
         | Java       | Conga     | JavaContract       | 0.0.1   |
+
+    @opsToolsFabric
+    Scenario Outline: Submit a transaction for a smart contract (OpsTool environment)
+        Given an environment 'myOpsToolsFabric' exists
+        And the wallet 'opsToolsWallet' with identity 'Org1MSPAdmin' and mspid 'org1msp' exists
```

Comment:
you need to add all the identities to the wallet or the setup step won't work correctly